clients/python/src/kaskada/utils.py

```diff
@@ -98,19 +98,6 @@ def handleGrpcError(rpc_error: grpc.Call):
     raise Exception(errorMessage) from None
 
 
-def to_uri(file: str) -> str:
-    # Remote file stores are prefixed appropriately.
-    if (
-        file.startswith("s3://")
-        or file.startswith("gs://")
-        or file.startswith("https://")
-    ):
-        return file
-    # TODO: Verify that the object actually exists locally
-    path = Path(file).absolute()
-    return f"file://{path}"
```

Comment:
unless I'm mis-understanding the request, I think we should still keep this functionality